.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ jobs:
             - v1-30-dependencies-
       - run:
           name: Install Dependencies
-          command: yarn
+          command: yarn --network-concurrency 1 install
```

Comment:
What does that do?

Reply:
it will set maximum number of concurrent network to one. Without `install dependencies` step was crashing in that PR

Reply:
I believe this is explained here why this could be necessary: https://github.com/yarnpkg/yarn/pull/2129

And it makes sense: we're fetching the code from your fork on GitHub right now.